components/IscnEditBar.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    :class="[
+      'flex',
+      'justify-end',
+      'items-center',
+      'w-full',
+      'pt-[24px]',
+      'gap-[8px]',
+      'lg:pt-0',
```

Comment:
If the padding-top is depends on parent, please inject it in parent

Reply:
[💄 Adjust UI of edit bar](https://github.com/likecoin/app-like-co/pull/438/commits/c5897943fcd96a2e796fa4f3c235a5bd7c26b00e)